client/src/bastionlab/polars/remote_polars.py

```diff
@@ -313,64 +315,161 @@ def join_asof(
             ),
         )
 
-    def histplot(self: LDF, col_x: str, col_y: str, bins: int = 10, **kwargs):
+    def histplot(
+        self: LDF, x: str = "count", y: str = "count", bins: int = 10, **kwargs
+    ):
+        """Histplot plots a univariate histogram, where one x or y axes is provided or a bivariate histogram, where both x and y axes values are supplied.
```

Comment:
So, if I understand correctly this doctoring, it is not raising any exception, correct?
Does Seaborn raise exceptions?